src/samples/UnAuthChildBenefitsClaim/index.tsx

```diff
@@ -549,10 +549,15 @@ export default function UnAuthChildBenefitsClaim() {
               sessionStorage.setItem('hasSessionTimedOut', 'true');
               clearTimer();
               deleteData();
-
-              setHasSessionTimedOut(false);
+              setHasSessionTimedOut(true);
             }
           }}
+          userTimeoutDelete={() => {
+            sessionStorage.setItem('hasSessionTimedOut', 'false');
+            clearTimer();
+            deleteData();
+            setHasSessionTimedOut(false);
```

Comment:
Why do we manage state in session and react state?

But everything else looks okay

Reply:
Good spot. The react state is unnecessary so I've removed it. Keeping the session state for if a user refreshes.